code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -815,6 +815,28 @@
 	X.recurring_injection(A, /datum/reagent/toxin/xeno_neurotoxin, XENO_NEURO_CHANNEL_TIME, XENO_NEURO_AMOUNT_RECURRING)
 
 
+//Panther Neurotox Sting
+/datum/action/xeno_action/activable/neurotox_sting/panther
+	name = "Panther Neurotoxin Sting"
+	mechanics_text = "A channeled melee attack that injects the target with neurotoxin over a few seconds, temporarily stunning them."
+	ability_name = "panther neurotoxin sting"
+	cooldown_timer = 50 SECONDS
+	plasma_cost = 60
+	keybind_signal = COMSIG_XENOABILITY_NEUROTOX_STING
+
+/datum/action/xeno_action/activable/neurotox_sting/panther/use_ability(atom/A)
+	var/mob/living/carbon/xenomorph/X = owner
+
+	succeed_activate()
+
+	add_cooldown()
+
+	GLOB.round_statistics.panther_neurotoxin_stings++
+	SSblackbox.record_feedback("tally", "round_statistics", 1, "panther_neurotoxin_stings")
+
+	X.recurring_injection(A, /datum/reagent/toxin/xeno_neurotoxin, XENO_NEURO_CHANNEL_TIME, XENO_NEURO_AMOUNT_RECCURING_PANTHER)
```

Comment:
Make it call the parent and store the differences in variables instead of fully redefining the proc.
Make generic behavior.